apps/pl-default/pattern-lab/_patterns/02-molecules-demo/utchero/utchero~lightflipped.yml

```diff
@@ -0,0 +1,22 @@
+utchero_1:
+  hero_image_select:
+    variant: 'primary'
+    src: ''
+    alt: 'Graybox placeholder image'
+    style: 'card'
+    placeholder: true
+  hero_title: 'Choose UTC flipped'
+  hero_text:
+    'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptatibus
+    quia, nulla! Maiores et perferendis eaque, exercitationem praesentium nihil.'
+  hero_button_text: 'Learn more'
+  hero_button_url: 'utc.edu'
+  hero_color: 'red'
```

Comment:
I know technically it doesn't matter if its 'red' because this is operating on an else statement in the template file...it just threw me off for a second. Should we call it light blue so future Brittany doesn't get caught in this loop of confusion when she forgets why this was 'red' and not red. ;)

Reply:
yeah, that's just a placeholder for something else. We need to update that one.